docs/2101UserGuide.md

```diff
@@ -321,69 +324,104 @@ List of commands that can be undone:
 
 #### Redoing a command: `redo`
 
-Redoes a command that the user has undone previously.
+Accidentally used `undo`? Don’t worry, you can redo your undone commands to retrieve your previous data too!
```

Comment:
Same nit as below, can consider "recover" instead of "retrieve"
```suggestion
Accidentally used `undo`? Don’t worry, you can redo your undone commands to recover your previous data too!
```

Or simply omit it entirely:
```suggestion
Accidentally used `undo`? Don’t worry, you can redo your undone commands too!
```

Reply:
I will be omitting it